### ISSUE TITLE
`reportUnneccessaryTypeIgnoreComment` should not report lines that are entirely comments

### ISSUE THREAD

**Issue by DetachHead** [DetachHead/basedpyright#714]
```py
# asdf # pyright:ignore[reportUndefinedVariable]
```
but if the line has any code along with the comment, it should still report it:

```py
foo = 1 # asdf # pyright:ignore[reportUndefinedVariable]
```
